### ISSUE TITLE
Pipe symbol not supported for python 3.8

### ISSUE THREAD

**Issue by merseneries** [archio1/DetectFakeNews#6]
https://github.com/archio1/DetectFakeNews/blob/b5ee81f7bc0227ca40bdb45d31322f263c67382f/src/FakeClass.py#L60

Please replace it with `Union`:
```python
from typing import Union
...
...
    def initialization_vector(self) -> Union[TfidfVectorizer, Tokenizer]:
```

**Comment by merseneries** [comment]
Fixed with commit 693a367af1f4e0dddaf212ccde1320b7953643f4

**Comment by merseneries** [comment]
Fixed with commit 693a367af1f4e0dddaf212ccde1320b7953643f4
